riscv/mmu.h

```diff
@@ -184,27 +184,32 @@ class mmu_t
       } \
   }
 
+  #define amo_or_zicbom_op(BODY) \
```

Comment:
Do you think it would be more understandable if this were named for what it does instead of what it's for? `convert_load_traps_to_store_traps()`?

Reply:
OK